### PR TITLE
Migration of cloud integration tests from Screwdriver to GitHub Actions

### PR DESCRIPTION
**Pull request by glebashnik**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

The following pipelines are migrated:
- integration-cloud
- integration-cloud-token
- integration-cloud-vector-search

### CODE REVIEW

File: .github/workflows/integration-cloud-token.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          pip install -e .[dev]
+      - name: Run integration tests
+        env:
+          VESPA_TEAM_API_KEY: ${{ secrets.VESPA_TEAM_API_KEY }}
+          CO_API_KEY: ${{ secrets.VESPA_CLOUD_SECRET_TOKEN }}
+          SD_ZIP_ARTIFACTS: true
```
Comment (thomasht86) [inline .github/workflows/integration-cloud-token.yml:32]: `SD_ZIP_ARTIFACTS` is [Screwdriver-specific](https://docs.screwdriver.cd/user-guide/environment-variables), so we probably won't need it.

File: .github/workflows/integration-cloud-vector-search.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          pip install -e .[dev]
+      - name: Run integration tests
+        env:
+          VESPA_TEAM_API_KEY: ${{ secrets.VESPA_TEAM_API_KEY }}
+          CO_API_KEY: ${{ secrets.VESPA_CLOUD_SECRET_TOKEN }}
+          SD_ZIP_ARTIFACTS: true
```
Comment (thomasht86) [inline .github/workflows/integration-cloud-vector-search.yml:32]:
```suggestion
```

File: .github/workflows/integration-cloud-token.yml (hunk 1 above)
Comment (thomasht86) [reply .github/workflows/integration-cloud-token.yml:32]:
```suggestion
          VESPA_TEAM_API_KEY: ${{ secrets.VESPA_TEAM_API_KEY }}
          VESPA_CLOUD_SECRET_TOKEN: ${{ secrets.VESPA_CLOUD_SECRET_TOKEN }}
          VESPA_CLIENT_TOKEN_ID: ${{ secrets.VESPA_CLIENT_TOKEN_ID}}
```

Comment (thomasht86) [reply .github/workflows/integration-cloud-token.yml:32]: `CO_API_KEY` is the api-key for cohere.com, which is only used in 1 or 2 of the notebooks.

Comment (glebashnik) [reply .github/workflows/integration-cloud-token.yml]: Makes sense, thanks for the info.

**Review by thomasht86 — APPROVED** [review]
Great! 🙌

**Pull request merged**